### ISSUE TITLE
Class name is not set correctly

### ISSUE THREAD

**Issue by sntran** [tanem/svg-injector#30]
When injecting, the resulting SVG has `class` attribute as `class="[object Set]"`. This can be seen on [Basic Usage example](https://codesandbox.io/s/github/tanem/svg-injector/tree/master/examples/basic-usage) on Chrome 71.0.3578.98 64-bit on macOS Mojave v10.14.12.

Screenshot for reference:

![image](https://user-images.githubusercontent.com/973206/51546881-be333f00-1e2a-11e9-849b-c088d2803f45.png)

**Comment by sntran** [comment]
It looks like the following TypeScript was not converted correctly:

https://github.com/tanem/svg-injector/blob/c173bbd9388334899e073c6b07b3460acb398c2f/src/inject-element.ts#L73-L79

The ES version has the following:

```es6
var mergedClasses = [].concat(
  new Set(
    [].concat(
      (svg.getAttribute('class') || '').split(' '), 
      ['injected-svg'], 
      (el.getAttribute('class') || '').split(' ')
    )
  )
).join(' ').trim();
```

That results in an array with the `Set` as its only element.

**Comment by sntran** [comment]
I was trying to submit a PR but unfortunately the tests all passed locally so I cannot reproduce it. My changes would be replacing spread operator with `Array.from`, i.e.:

```es6
const mergedClasses = Array.from(
  new Set([
    ...(svg.getAttribute('class') || '').split(' '),
    'injected-svg',
    ...(el.getAttribute('class') || '').split(' ')
  ])
)
  .join(' ')
  .trim()
```

**Comment by tanem** [comment]
@sntran thanks for this report, good catch 🙏 You were on the right track, I just created a PR with the same change actually.

PS sorry about that test situation. This library is in the middle of a big refactor, and I've been getting caught up in my day job a bit lately, so haven't been able to dedicate as much time to getting this refactor done as I'd like ⏳

**Comment by tanem** [comment]
@sntran just published #31 as `v5.0.5` if you want to try that version out now?

**Comment by sntran** [comment]
It works great! Thanks for taking the time to fix it so quickly.
